### PR TITLE
Close #189 - Junit cmd CI

### LINKED ISSUE #189: Add proper command line for junit in the pipeline
(Use the one of the Pharo build)

### PR DESCRIPTION
**Pull request by VincentBlondeau**
Fix #189

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -42,7 +42,7 @@ def buildArchitecture(architecture, pharoVersion) {
 		    }
     	  stage("Test Pharo${pharoVersion}-${architecture}-bits") {
 		    	sh "VERSION=$commitHash ./build.sh test"
-			    junit '*.xml'
+			    junit allowEmptyResults: true, testResults: '*.xml'
```
Comment (jecisc) [inline Jenkinsfile]: Why allowing empty results? 

If we know there are tests then we should not allow empty results. If there are empty results then it means there was a problem during loading/testing.

Comment (VincentBlondeau) [reply Jenkinsfile]: Right

**Comment by guillep** [comment]
I don't quite understand why this is needed. 
```
junit allowEmptyResults: true, testResults: '*.xml'
```

and 

```
junit '*.xml'
```
are equivalent, since the default parameter of the junit step is the path to the xml files...

Just see the development branch, its showing the test graph

https://ci.inria.fr/pharo-ci-jenkins2/job/PharoLauncher-Pipeline/job/development/

screen shot 2018-07-07 at 11 35 14

So, is this issue still valid?

**Comment by VincentBlondeau** [comment]
At first I saw that the test list were not displaying, so I was thinking it was due to this. But at end, it appears. So we can close this issue as far as the 2 cmds are the them

**Pull request closed without merging**